lib/renderer.js

```diff
@@ -132,7 +132,11 @@ module.exports = function(data, options) {
     }
   });
 
+  // exec filter to extend renderer.
+  const renderer = new Renderer();
+  this.execFilterSync('marked:renderer', renderer, {context: this});
+
   return marked(data.text, Object.assign({
-    renderer: new Renderer()
+    renderer: renderer
```

Comment:
```suggestion
    renderer
```

Short hand object syntax should be used.